earth_enterprise/BUILD_RHEL_CentOS.md

```diff
@@ -15,6 +15,8 @@ sudo yum install -y epel-release
 
 ```bash
 sudo yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+sudo yum-config-manager --enable rhel-7-server-rhui-optional-rpms
```

Comment:
I think we need different sections here that indicate what the commands are on regular machines vs on AWS machines.

Reply:
Okay, I didn't realize there were different versions for AWS and non-AWS.
Is non-AWS the same as what was there before?  I'll see what I can find out.

Reply:
As far as I know, this is what's needed for non-AWS machines:
```
 sudo subscription-manager repos --enable=rhel-7-server-optional-rpms
 sudo subscription-manager repos --enable=rhel-7-server-extras-rpms
```
but I haven't tested that recently.